changelogs/fragments/47182-os_port_order_difference_should_not_trigger_changes.yml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - os_port does no longer trigger change when allowed_address_pairs items is in different order
```

Comment:
```suggestion
  - "os_port - no longer triggers change when ``allowed_address_pairs`` items are in different order."
```

Reply:
Thanks, I always get a bit confused with the changelog formatting, will fix